web/src/user_group_popover.ts

```diff
@@ -52,22 +52,32 @@ export function handle_keyboard(key: string): void {
     popover_menus.popover_items_handle_keyboard(key, $items);
 }
 
-// element is the target element to pop off of;
-// message_id is the message id containing it, which should be selected;
+// Function called with `message_id` when user-group-mention is clicked.
+// Function called with user_group object when input_pill is clicked.
+// `element` is the target element to pop off of;
+// `message_id` is the message id containing it, which should be selected;
 export function toggle_user_group_info_popover(
     element: ReferenceElement,
-    message_id: number,
+    group_or_message: number | user_groups.UserGroup,
```

Comment:
Do we need this change? We would anyway pass `element` and that can be used to get the group object and `message_id` can just be passed as undefined.